### ISSUE TITLE
Irregular HTTP error code: 404

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bogenpirat/remote-torrent-adder#141]
```
What version of Chrome are you using? On what operating system?
I'm using Version 41.0.2272.118 (64-bit) on OS X 10.10.2

What client (version?) are you connecting to? Are you using SSL?
I'm connecting to BitTorrent 7.9.2 not using SSL

What steps will reproduce the problem?
1.Client install, Web UI configuration
2. Remote Torrent Adder configuration (needless to say, same as client)
3. Trying to add a magnet link or opening a .torrent file

What is the expected output? What do you see instead?
I assume that adding the torrent to the list. I see the attached popup

```

Original issue reported on code.google.com by `agus...@outlook.com` on 7 Apr 2015 at 3:00

**Comment by xor-gate** [comment]
I have exactly the same problem only with ruTorrent:
Chrome Version 47.0.2526.106 (64-bit)
ruTorrent v3.7 (proxied behind nginx)
It seems to add the torrent as expected but then tries to navigate to "/"

**Comment by xor-gate** [comment]
Here is a screenshot:
![untitled](https://cloud.githubusercontent.com/assets/1050166/12335474/a818ab06-baff-11e5-906e-4dc67a15178d.png)

**Comment by benjamincharity** [comment]
Also experiencing this with ruTorrent...

Chrome: 47.0.2526.106 (64-bit)
ruTorrent: 3.6

**Comment by zammtech** [comment]
I had this problem and types /rutorrent/ in the "Relative path" textbox and now the torrents add fine.

**Comment by vurtomatic** [comment]
I'm experiencing this too. Torrents add fine but getting the 404 error.

**Comment by pbrink231** [comment]
I am also getting this error.

The torrents do not add.  This happens on and off.  If I try again tomorrow it might work.  so there are extended periods of time it doesnt work.  I cannot link it anything at the moment.  I change nothing between working and not working.

![rta_error_0](https://cloud.githubusercontent.com/assets/8224560/24717350/28017c16-1a00-11e7-9053-d793b4f7cdb3.PNG)

I have restarted rtorrent, waited and checked and it does not solve the issue.

Chrome
57.0.2987.133 (64-bit)

rTorrent (ruTorrent) 0.9.6/0.13.6

**Comment by bogenpirat** [comment]
self-signed certificate?

**Comment by pbrink231** [comment]
no.  the ruTorrent is hosted on seedboxes.cc

Sometimes it works.  sometimes it doesnt.  I will work for a while then randomly stop.  Wont work for a while then ill come back later and it will work again.

**Comment by bogenpirat** [comment]
off the bat that sounds to me like it might be an issue with the host and not the extension.

two things:

- when it does happen, check the background page view (`chrome://extensions/?id=oabphaconndgibllomdcjbfdghcmenci`, enable dev mode top right, click on background page) and report back any errors it might show there
- can you provoke it happening consistently (i.e. it never works) when you completely close chrome, and then restart it? to completely close chrome, you'll have to quit it from the tray icon in the lower right (if you're on windows).

**Comment by pbrink231** [comment]
I tried but this was one of the times it was succesful.

Now I know where to look when it breaks and will come back with more detailed info when it fails again.

BTW i got this message on a succesful attempt
![image](https://cloud.githubusercontent.com/assets/8224560/24803638/cb51911a-1b79-11e7-86a6-6944ecb01730.png)

**Comment by pbrink231** [comment]
Here is what i get when i receive the error

![image](https://cloud.githubusercontent.com/assets/8224560/24830815/236b5b60-1c5b-11e7-9dae-645a791c5d61.png)

Hopefully this helps.  Looks like it is net::ERR_SPDY_PROTOCOL_ERROR

This is the place it brings me to

XMLHttpRequest.prototype.sendAsBinary = function(datastr) {
	function byteValue(x) {
		return x.charCodeAt(0) & 0xff;
	}
	var ords = Array.prototype.map.call(datastr, byteValue);
	var ui8a = new Uint8Array(ords);
	this.send(ui8a.buffer); <--- This is the line that fails
}

**Comment by aztazt** [comment]
Exact same issue with ruTorrent on seedboxes.cc.

**Comment by tacosuzy** [comment]
Is this extension compatible with pulsedmedia box rtorrent and stable chrome? I am getting this error with several iterations of info entered. I am basically guessing due to no tutorial with this box. The interface is multi tab all on the same page. the URL is as this *.pulsedmedia.com/user-myusername/ The star is there in place of the server numbers. I was able to make a connection using Xirvik .torrent to seedbox uploader extension test feature by adding http:// in front of the url but get an error when clicking torrent and magnet. Anyone know how to set remotetorrentadder up with Pulsedmedia? I have no idea what goes in relative path or host. I have asked support if they know but I figured I would ask here as well to save time.
